taskcluster/ci/beetmover/kind.yml

```diff
@@ -0,0 +1,35 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: mozillavpn_taskgraph.loader.multi_dep:loader
+
+transforms:
+    - mozillavpn_taskgraph.transforms.multi_dep:transforms
+    - mozillavpn_taskgraph.transforms.beetmover:transforms
+    - taskgraph.transforms.task:transforms
+
+group-by: build-type
+
+only-for-build-types:
+    - macos/opt
+
+kind-dependencies:
+    - signing
+
+primary-dependency: signing
+
+job-template:
+    worker-type: beetmover
+    run-on-tasks-for: [action, github-release]
+    bucket:
+        by-level:
+            "3": "release"
+            default: "dep"
```

Comment:
I'm not sure if setting this `by-level` makes sense. Only a `github-release` or action should be using the `release` bucket, so a regular push should use the `dep` bucket even though it is still level 3. It's not a big deal as long as we have the scopes configured correctly though. Maybe if someone manages to run a BM on a regular push, a scope error would be a good thing?

Reply:
Right, I just mean should we add even more logic here? E.g, it's `release` if level 3 *and* `tasks_for in ["github-release", "action"]`. Would probably need to set it in a transform at that point unless you wanted nested `by-` statements.